src/components/Vuetify/mixins/breakpoint.ts

```diff
@@ -1,90 +1,62 @@
 import Vue from 'vue'
 import { VuetifyBreakpoint } from 'types'
 
+const breakpoints = {
+  xs: 600,
+  sm: 960,
+  md: 1280 - 16,
+  lg: 1920 - 16,
+  xl: Infinity
+}
+const keys = Object.keys(breakpoints) as (keyof typeof breakpoints)[]
```

Comment:
Use the `keys` function from util/helpers

Reply:
Why don't you just override the definition from `lib.es5.d`?
Just doing an
```ts
// index.d.ts
interface ObjectConstructor {
  keys<O extends object>(o: O): (keyof O)[];
  keys<O>(o: O): [];
}
```
works for me I think?